wasmedge.yaml

```diff
@@ -0,0 +1,69 @@
+package:
+  name: wasmedge
+  version: 0.13.3
+  epoch: 0
+  description: WasmEdge is a lightweight, high-performance, and extensible WebAssembly runtime for cloud native, edge, and decentralized applications.
+  target-architecture:
+    - all
```

Comment:
```suggestion
```

Reply:
We've got quite a few packages that explicitly set 
```
  target-architecture:
    - all
```

Let's merge this and we can cleanup all references to the above in a followup.